brave/brave-core
# Auto-excluded publishers now show properly in Rewards panel

Fixes brave/brave-browser#1467

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:

1. Open Brave with clean profile and enable Rewards
2. Visit an auto-excluded site for the minimum time normally required to add sites to auto-contribute
3. Open Rewards panel and verify that site specific information is shown
4. Open Rewards page and verify that publisher does not show in auto-contribute

Additional information:

NOTE FOR QA:
This PR will not work across browser restarts until https://github.com/brave-intl/bat-native-ledger/pull/131 is merged. If Brave is restarted, auto-excluded sites will show in auto-contribute. Use clean profile when testing this PR.


## Reviewer Checklist:
